staging/src/k8s.io/client-go/OWNERS

```diff
@@ -1,45 +1,15 @@
 approvers:
 - caesarxuchao
 - deads2k
-- krousey
 - lavalamp
+- liggitt
 - smarterclayton
 - sttts
-- liggitt
 reviewers:
-- thockin
-- lavalamp
-- smarterclayton
-- wojtek-t
-- deads2k
-- yujuhong
-- derekwaynecarr
 - caesarxuchao
-- vishh
-- mikedanese
+- deads2k
+- lavalamp
 - liggitt
-- nikhiljindal
-- gmarek
-- erictune
-- davidopp
-- pmorie
+- smarterclayton
```

Comment:
Remove clayton from reviewers, but leave as approver. 

Add @soltysh as a reviewer (not approver).

Reply:
done